adapters/src/test/java/com/squareup/moshi/adapters/PolymorphicJsonAdapterFactoryTest.java

```diff
@@ -27,17 +30,17 @@
 import okio.Buffer;
 import org.junit.Test;
 
-import static org.assertj.core.api.Assertions.assertThat;
-import static org.junit.Assert.fail;
-
 @SuppressWarnings("CheckReturnValue")
 public final class PolymorphicJsonAdapterFactoryTest {
-  @Test public void fromJson() throws IOException {
-    Moshi moshi = new Moshi.Builder()
-        .add(PolymorphicJsonAdapterFactory.of(Message.class, "type")
-            .withSubtype(Success.class, "success")
-            .withSubtype(Error.class, "error"))
-        .build();
+  @Test
+  public void fromJson() throws IOException {
+    Moshi moshi =
+        new Moshi.Builder()
+            .add(
```

Comment:
ew

Reply:
you get used to it

https://github.com/google/google-java-format/wiki/The-Rectangle-Rule